it.unibo.tuprolog.ide.yml

```diff
@@ -0,0 +1,37 @@
+id: it.unibo.tuprolog.ide
```

Comment:
does upstream have control over tuprolog.unibo.it ? It seems to redirect to somewhere else.

Reply:
Yup, it's just a redirect to the wiki used to document the project